subcommands/targets/deltas.go

```diff
@@ -0,0 +1,109 @@
+package targets
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	byTag  string
+	dryRun bool
+)
+
+func init() {
+	var deltas = &cobra.Command{
+		Use:   "static-deltas <target-version> [<from-version>...]",
+		Short: "Generate static deltas to the given target version to make OTAs faster",
+		Run:   doDeltas,
+		Args:  cobra.MinimumNArgs(1),
+		Long: `In many cases OTAs will have many ostree changes. These OTAs can be 
```

Comment:
I am not sure about it, just think that `OTAs` should be `OTA updates` since OTA stands just for Over-The-Air?

Reply:
i fear we probably say "OTAs" or just OTA in many places now. After a big of googling i'll say Android is probably the best to follow:

 https://source.android.com/devices/tech/ota

they say "OTA updates". I'll fix this to not make things worse, but there's probably a lot of stuff docs/blogs/code that repeats this.

Reply:
https://github.com/foundriesio/fioctl/pull/99/commits/abb98ca62cfc1ce83520149e69977c0ff7656ee8